lib/util/get-convert-path.js

```diff
@@ -6,6 +6,15 @@
 
 const { Minimatch } = require("minimatch")
 
+/**
+ * @typedef {Object} PathConvertion
+ * @property {string[]} include
+ * @property {string[]} exclude
+ * @property {[string, string]} replace
+ * @typedef {PathConvertion[] | Record<string, [ string, string ]>} ConvertPath
+ * @typedef {{match: (filePath: string) => boolean, convert: (filePath: string) => string}} Converter
+ */
```

Comment:
I would hive every `@typedef` its own comment block

```suggestion
 * @property {[string, string]} replace
 */

 /** @typedef {PathConvertion[] | Record<string, [ string, string ]>} ConvertPath */
 /** @typedef {{match: (filePath: string) => boolean, convert: (filePath: string) => string}} Converter */
```

Reply:
:+1: this is a good point!